src/status_im/contexts/wallet/swap/swap_proposal/view.cljs

```diff
@@ -12,4 +12,8 @@
      [quo/button
       {:on-press #(rf/dispatch [:show-bottom-sheet
                                 {:content slippage-settings/view}])}
-      (str "Edit Slippage: " max-slippage "%")]]))
+      (str "Edit Slippage: " max-slippage "%")]
```

Comment:
should be translated no?

Reply:
It is a temporary button to launch Edit slippage drawer, this will completely removed when we implement Swap Input screen and do a proper integration with router